### PR TITLE
Fixes apply debug console font size setting to input field

### LINKED ISSUE #90714: Apply debug console font size setting to input field
It's possible to adjust the debug console's font size via

```json
"debug.console.fontSize": 12,
```

This currently does not affect the input field as is obvious when cranking up the size to 20. Since the input is part of the debug console, I think it should though. 

large-font

### PR DESCRIPTION
**Pull request by jeanp413**
This PR fixes #90714

### CODE REVIEW

**Review by AdrianDeAnda — APPROVED** [review]

File: src/vs/workbench/contrib/debug/browser/repl.ts [hunk 1]
```diff
@@ -305,6 +305,16 @@ export class Repl extends ViewPane implements IHistoryNavigationWidget {
 			`;
 
 			this.tree.rerender();
+
+			this.replInput.updateOptions({
+				fontSize,
```
Comment (isidorn) [inline src/vs/workbench/contrib/debug/browser/repl.ts:310]: Why not take the `fontFamily`, `lineHeight` computed above? Why use `debugConsole.`

**Comment by isidorn** [comment]
@jeanp413 great PR as usual, thank you very much for creating this.
I have left a comment in the code changes. Also I tried this out and seems to work quite well, noticed one issue though: for larger font sizes the input chevron is not focused. We probably hard code the size somewhere, and the font change breaks this. Can you maybe look into improving the centering of the chevron on the left?

Screenshot 2020-04-06 at 13 03 09

File: src/vs/workbench/contrib/debug/browser/repl.ts (hunk 1 above)
Comment (jeanp413) [reply src/vs/workbench/contrib/debug/browser/repl.ts:310]: Those values are for the inline css
- `lineHeight` computed above has type `string` (``? `${debugConsole.lineHeight}px` : '1.4em'``) and the one needed for `updateOptions` must be `number`
- `fontFamily` computed above resolves to `'var(--monaco-monospace-font)'` when the default font is requested

Comment (isidorn) [reply src/vs/workbench/contrib/debug/browser/repl.ts:310]: Makes sense. Thank you

**Comment by isidorn** [comment]
@jeanp413 your comment makes sense. Once we resolve the centering of the chevron (if possible) I would be open to merge in this PR.

**Comment by jeanp413** [comment]
Pushed some changes to center de input chevron, let me know if the css is ok

**Comment by isidorn** [comment]
@jeanp413 Yeah, css looks great. Let's merge this.
Thanks a lot 🍻 ☀️

**Pull request merged**